ui/lib/apps/KeyViz/components/KeyViz.tsx

```diff
@@ -13,12 +13,12 @@ import KeyVizToolbar from './KeyVizToolbar'
 
 import './KeyViz.less'
 
-type CacheEntry = {
-  metricType: DataTag
-  dateRange: number
-  expireTime: number
-  data: HeatmapData
-}
+// type CacheEntry = {
+//   metricType: DataTag
+//   dateRange: number
+//   expireTime: number
+//   data: HeatmapData
+// }
```

Comment:
Why do we have these changes?

Reply:
This type is unused (reported by eslint), we can remove it.